build-config/resources/Ballerina.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+org = "ballerinax"
+name = "aws.redshiftdata"
+version = "@toml.version@"
+authors = ["Ballerina"]
+keywords = ["Data Warehouse", "Columnar Storage", "Cost/Paid", "vendor/aws"]
+repository = "https://github.com/ballerina-platform/module-ballerinax-aws.redshiftdata"
+#icon = "icon.png"
```

Comment:
Shall we add the `icon` as well ? We can reuse the same icon we use for Redshift JDBC connector. @aashikam FYI

Reply:
https://github.com/ballerina-platform/module-ballerinax-aws.redshift/blob/main/ballerina/icon.png

Reply:
Sure

Reply:
It seems like the `icon.png` has not been added to the repo, why have we marked this as resolved ? @chathushkaayash

Reply:
https://github.com/ballerina-platform/module-ballerinax-aws.redshiftdata/blob/7eacb5b566344dc9c299f05e6d56b8544870854e/ballerina/icon.png

Added